usaspending_api/awards/v2/filters/matview_transaction.py

```diff
@@ -115,7 +115,7 @@ def transaction_filter(filters):
                     'time_period',
                     'award_type_codes',
                     'agencies',
-                    'legal_entities',
```

Comment:
In the interest of not breaking anything, could we keep the existing filter inplace and add the new search_text field? Are we 100% sure nothing will use `legal_entities` after the website PR?

Reply:
I would think we shouldn't allow it going forward. @tony-sappe if we're not 100% sure, then I agree, we should keep it.

Reply:
Keep the filter.